src/vs/workbench/contrib/preferences/browser/media/settingsWidgets.css

```diff
@@ -29,15 +28,24 @@
 .settings-editor > .settings-body > .settings-tree-container .setting-item.setting-item-list .setting-list-sibling {
 	max-width: 10%;
 }
-.settings-editor > .settings-body > .settings-tree-container .setting-item.setting-item-list .setting-list-object-widget .setting-list-object-key,
+.settings-editor > .settings-body > .settings-tree-container .setting-item.setting-item-list .setting-list-object-widget .setting-list-object-key {
+	min-width: 40%;
+}
+.settings-editor > .settings-body > .settings-tree-container .setting-item.setting-item-list .setting-list-object-widget .setting-list-object-key {
+	flex: 2;
```

Comment:
What difference exactly does the flex 2/3 make? I think it would be best if the two columns were the same width, but I guess you have to compensate for the ok/cancel buttons.

Reply:
flex 2/3 makes key width 40% and value width 60%, and as you mentioned, I did it to compensate for the ok/cancel button